string object_action
string object_name
float32[] euler_pose

